source/community/release_notes/redwood.rst

```diff
@@ -25,10 +66,140 @@ Instructor Experiences
+
+   -  To create and populate the search index, you must run a one-time
+      command from the CMS shell:
+      ``python manage.py cms reindex_studio --experimental``. This
+      command may take a while if you have a lot of courses and/or
+      libraries in Studio; it will display regular progress indicators
+      while it is running. We are interested in hearing how long it
+      takes for you - please share your experience (see next bullet).
+      This command reads from MySQL/MongoDB but does not write to them;
+      it only writes to Meilisearch. Once the indexing has completed, it
+      should not be necessary to run it again; from that point forward,
+      the indexes will be updated automatically as needed.*
+
+   -  Please share your feedback about Meilisearch, indexing, and
+      operations in `this Discourse thread <https://discuss.openedx.org/t/is-meilisearch-a-viable-upgrade-alternative-to-opensearch/12400>`_ or the `#ops <https://openedx.slack.com/archives/C08B4LZEZ>`_ Slack channel. Please share feedback about
+      the new course search feature in general `in the discussion forums <https://discuss.openedx.org/t/feedback-thread-new-course-search/13076>`_ or in the `#wg-product-core <https://openedx.slack.com/archives/C057J2D1WU9>`_ Slack channel.*
+
 
 Deprecations & Removals
 ***********************
 
+-  *Badges app has been deprecated and removed from ``edx-platform``.
+   See `[DEPR]: lms/djangoapps/badges <https://github.com/openedx/edx-platform/issues/31541>`_ .*
+
+   -  *``accomplishments_shared`` field is removed from payloads and
+      settings*
+
+   -  *``ENABLE_OPENBADGES`` is no longer available for configuration in
+      ``FEATURES``*
+
+   -  *``BADGING_BACKEND``, ``BADGR_BASE_URL``, ``BADGR_ISSUER_SLUG``,
+      ``BADGR_USERNAME``, ``BADGR_PASSWORD``,
+      ``BADGR_TOKENS_CACHE_KEY``, ``BADGR_TIMEOUT``,
+      ``BADGR_ENABLE_NOTIFICATIONS`` are also not configurable anymore.*
+
+-  * In edxapp, the Waffle switch ``ip.legacy`` is removed. See `[DEPR]: legacy_ip code and Waffle switch <https://github.com/openedx/edx-platform/issues/33733>`_ .*
+
+   -  *Any deployment that has been relying on this legacy IP address
+      option will need to switch to setting
+      ``CLOSEST_CLIENT_IP_FROM_HEADERS`` appropriately. See `Nutmeg Announcement <https://openedx.atlassian.net/wiki/spaces/COMM/pages/3205201949>`_ for
+      details.*
+
+-  *Asset Processing (webpack, collectstatic, etc.) using Paver Commands in edx-platform is now Deprecated and will not be available in Sumac*
+   - `[DEPR]: Asset processing in Paver <https://github.com/openedx/edx-platform/issues/31895>`_.
+   -  *This should cover everything that a site operator needs to do as a part of deployment.*
+   -  *Non deployment paver commands will be removed by Sumac*
+
+-  *The django-splash capability was removed from edx-platform and the
+   relevant code has been archived.*
+   - https://github.com/openedx/public-engineering/issues/224
```

Comment:
Do we have a section for "known issues" where we can include some of the issues found during testing that we didn't get to close before the release?

Reply:
We don't but we should! I'll add it with my next revision.

Here's the thing though: Please tell me it'll be easy for me to get links to those, too much of the rest of the information gathering for this is like tooth extraction :)

Reply:
I'm not sure what folks think but I believe using this list is enough: https://github.com/orgs/openedx/projects/28/views/16

Reply:
Those are known issues from "release testing", they're not filtered by redwood or any other release, so we should add a clarification note if it's gonna be used as is.